tests/containers/integration-test-snapshot

```diff
@@ -19,4 +23,13 @@ RUN dnf install \
 
 RUN dnf -y clean all
 
+RUN if [ "$with_coverage" = "1" ]; then \
+        dnf install -y --repo bluechi-rpms \
+        --repofrompath bluechi-rpms,file:///tmp/bluechi-rpms/ \
+        --nogpgcheck \
+        --nodocs \
+        bluechi-coverage && \
+    dnf -y clean all; \
+    fi
+
```

Comment:
This is wrong, in `integration-test-snapshot` container we need to install everything from bluechi-snapshot repo, we are not using any local repo in this container